app/Config/Filters.php

```diff
@@ -4,6 +4,7 @@
 
 class Filters extends BaseConfig
 {
+
```

Comment:
Please remove this newline, as it constitutes a change to a file in **app** that otherwise does not need to be touched.
```suggestion
```